lua/py_lsp/init.lua

```diff
@@ -215,10 +220,11 @@ M.py_run = function(...)
 end
 
 M.setup = function(opts)
+
     -- Init all commands
     for command, func in pairs(commands.commands) do
         vim.api.nvim_create_user_command(command, M[func], {
-            desc = commands.commands_to_text[command]
+            desc = commands.commands_opts[command]
```

Comment:
The third parameter here should just be `commands.commands_opts[command]` because commands_opts contains more than just desc now.